faf-commons-data/src/main/java/com/faforever/commons/replay/ReplayDataParser.java

```diff
@@ -51,12 +53,12 @@ public ReplayDataParser(Path path) {
   }
 
   private String readString(LittleEndianDataInputStream dataStream) throws IOException {
-    StringBuilder stringBuilder = new StringBuilder();
-    char character;
-    while ((character = (char) dataStream.readByte()) != 0) {
```

Comment:
Do i understand right, that here we read each character as single byte? So there is not possible to have utf multi-byte characters?

Reply:
Yes, as example russian symbols looks like 110XXXXX 10XXXXXX so they dont dispay correctly ib replay details window.